README.md

```diff
@@ -275,3 +275,19 @@ If the user does not wish to copy the role in the `roles` folder of this reposit
 export ANSIBLE_ROLES_PATH=<the-dir-where-the-role-is-located>
 . . . (run deployment as you would normally do)
 ```
+
+### How to rewrite an ansible task to be softfailed automatically in OSADO
```

Comment:
### Mark temporary workaround for known issues

If Ansible code is added to temporary workaround known issue, already associated to an open ticket, there's a convention to communicate it to opneQA. Use `ansible.buildin.debug` with a specific format.